tritium-annotations/build.gradle

```diff
@@ -9,6 +9,7 @@ dependencies {
     implementation project(':tritium-slf4j')
     implementation project(':tritium-tracing')
     implementation 'com.google.code.findbugs:jsr305'
+    implementation 'com.google.errorprone:error_prone_annotations'
```

Comment:
this might be a bit unfortunate bumping error-prone deps downstream now & in the future

Reply:
Perhaps compileOnly?

Reply:
It's from the `@CanIgnoreReturnValue` which has Classfile retention 

https://github.com/google/error-prone/blob/master/annotations/src/main/java/com/google/errorprone/annotations/CanIgnoreReturnValue.java#L36-L37

Reply:
will let https://github.com/palantir/gradle-baseline/pull/2352 bump to error-prone 2.15.0 first

Reply:
[gradle-baseline 4.155.0 includes error-prone 2.15.0](https://github.com/palantir/gradle-baseline/releases/tag/4.155.0)